src/renderer/features/extension-wallet/types.ts

```diff
@@ -18,7 +18,14 @@ export type SubWalletExtensionWallet = Wallet & {
   extension: ExtensionType;
 };
 
-export interface ExtensionAccount extends UniversalAccount {
+export type ExtensionAccount = ExtensionUniversalAccount | ExtensionChainAccount;
```

Comment:
Could we move this line after `export interface ExtensionChainAccount extends ChainAccount {`?
So it reads top down like a book 😄